app/src/main/java/mozilla/lockbox/view/RootActivity.kt

```diff
@@ -14,7 +14,7 @@ import mozilla.lockbox.presenter.RoutePresenter
 
 @ExperimentalCoroutinesApi
 class RootActivity : AppCompatActivity() {
-    private var presenter: RoutePresenter = RoutePresenter(this)
+    internal var presenter: RoutePresenter = RoutePresenter(this)
```

Comment:
See above re using standard `onCreate` and `onDestroy` methods in the `simulateAppStartup` method.